toplevel/ccompile.ml

```diff
@@ -92,18 +92,20 @@ let create_empty_file filename =
   let f = open_out filename in
   close_out f
 
+let check_pending_proofs () =
+  let pfs = Vernacstate.Declare.get_all_proof_names () [@ocaml.warning "-3"] in
+  if not (CList.is_empty pfs) then
+    fatal_error (str "There are pending proofs: "
+                 ++ (pfs
+                     |> List.rev
+                     |> prlist_with_sep pr_comma Names.Id.print)
+                 ++ str ".");
+  let pm = Vernacstate.Declare.get_program () [@ocaml.warning "-3"] in
+  Declare.Obls.check_solved_obligations ~what_for:(Pp.str "file") ~pm
```

Comment:
To make the error easier to understand, maybe change `Pp.str "file"` to the filename of the current file. Otherwise, given a long log generated by `make -j`, it’s impossible to tell at a glance which file has the error in it.